DigitalOffice/Tests/CompanyServiceTests/Validators/AddCompanyValidatorTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
```

Comment:
Using directive is not required by the code and can be safely removed

Reply:
Fixed.